tests/python_functional/functional_tests/destination_drivers/snmp_destination/general/test_snmp_destination_acceptance.py

```diff
@@ -39,6 +39,6 @@ def test_snmp_dest_acceptance(config, syslog_ng, snmptrapd, snmp_test_params):
 
     syslog_ng.start(config)
 
-    received_traps = snmptrapd.get_traps()
-    assert snmp_test_params.get_expected_basic_trap() == received_traps
-    assert any(snmp_test_params.get_default_community() in line for line in snmptrapd.get_raw_traps())
+    expected_traps = snmp_test_params.get_expected_basic_trap()
+    assert expected_traps == snmptrapd.get_traps(len(expected_traps))
+    # assert any(snmp_test_params.get_default_community() in line for line in snmptrapd.get_raw_traps())
```

Comment:
is this a leftover?

Reply:
Yes, this should be deleted. I will fix it.

Reply:
I have fixed this.